src/TouristInfoCards.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import "./TouristInfoCards.css";
+
+const Card = ({ image, title, description, link }) => {
+  return (
+    <div className="card">
+      <img src={image} className="card-img-top" alt={title} />
+      <div className="card-body">
+        <h5 className="card-title">{title}</h5>
+        <p className="card-text">{description}</p>
+        <a href={link} className="btn btn-primary">
+          More Information
+        </a>
+      </div>
+    </div>
+  );
+};
```

Comment:
This file src/TouristInfoCards.js contains 2  components, a Card component and a TouristInfoCards component

Generally it is best practice to have 1 component per file, https://github.com/airbnb/javascript/blob/master/react/README.md#basic-rules. 

Your Card compoent is a Pure Stateless component, so it is totally ok to have 2 components in 1 file here 

But it would still have it in `src/card ` because the Card component could be reused by other new components. Imagine that you want to create another React Component using the same Card component you have defined here.

Problem no.1 the card isn't being exported right now. If it was exported, all Card component imports would look like this

`import {Card} from ./TouristInfoCard`

instead of 

`import Card from ./Card`

Problem no.2 you would want to isolate each component in a single test file

Reply:
Thank you for explaining this so clearly. I'll remember this for future.